src/functionHandlerProvider/functionHandlerProvider.ts

```diff
@@ -2,4 +2,5 @@ import { FunctionConfiguration } from "../models/projectConfiguration.js";
 
 export interface FunctionHandlerProvider {
     getHandler(functionConfiguration: FunctionConfiguration): Promise<string>;
+    writeAdditionalFiles(path: string): Promise<void>;
```

Comment:
Does the naming of this function make sense for other Function Handler Providers like GCP or Azure? I feel like the naming of this can be more explicit.

Reply:
This is not particular to Lambda, it is just a way to generate additional files needed by the provider wrapper, if needed. If other providers don't need additional files, they don't need to do anything.

Reply:
I think the whole interface should be rewritten. Instead of having two methods one that returns the `getHandler` and one that `writeAdditionalFiles`, I think we should have just one method that is called `writeHandler` and does everything.